bundle/manifests/konveyor-operator.clusterserviceversion.yaml

```diff
@@ -8,8 +8,7 @@ metadata:
           "apiVersion": "tackle.konveyor.io/v1alpha1",
           "kind": "Tackle",
           "metadata": {
-            "name": "tackle",
-            "namespace": "konveyor-tackle"
```

Comment:
Is this intentional?

Reply:
It was but I'm not certain now why I did it. Will check this again.